socialhome/streams/app/main.js

```diff
@@ -0,0 +1,11 @@
+import Vue from "vue"
+import VueResource from "vue-resource"
+import "./stores/streamStore"
+import "./components/Stream.vue"
+
+Vue.use(VueResource)
+
+export default new Vue({
+    el: "#app",
+    http: {root: "/"}
```

Comment:
I guess it's fine if the library is already included? I'll work on the streams API next week 👍